src/__internal__/label/label.style.ts

```diff
@@ -24,12 +24,11 @@ const LabelStyle = styled.label`
     `}
 `;
 
-LabelStyle.propTypes = {
-  disabled: PropTypes.bool,
-  isRequired: PropTypes.bool,
-};
+const DEFAULT_CONTAINER_WIDTH = 30;
 
-export const StyledLabelContainer = styled.div`
+export const StyledLabelContainer = styled.div<
+  Pick<LabelProps, "align" | "inline" | "pr" | "pl" | "width" | "optional">
+>`
```

Comment:
what do you think about defining `StyledLabelContainerProps` here and then extending the component interface with it?

Reply:
See above